app/Events/StandAssignedEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\Stand\StandAssignment;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class StandAssignedEvent implements ShouldBroadcast
+{
+    const CHANNEL = 'stand-assignments';
+
+    /**
+     * @var StandAssignment
+     */
+    private $standAssignment;
+
+    public function __construct(StandAssignment $standAssignment)
+    {
+        $this->standAssignment = $standAssignment;
+    }
+
+    public function broadcastWith()
+    {
+        return [
+            'callsign' => $this->standAssignment->callsign,
```

Comment:
Do you want to use the getter here instead of the direct assignment? Or is the getter to help when it is typehinted?

Reply:
Sorry not sure what you mean here?

Reply:
You've got `$this->getStandAssignment()` - was questioning if you wanted to use a consistent interface for getting the stand assignment.

Reply:
Oh yes, gotcha, done!